region/brinstar/kraid.json

```diff
@@ -1631,6 +1691,44 @@
                   "note": "Wait to hit one of Mini-Kraid's spikes, then run through."
                 }
               ]
+            },
+            {
+              "id": 2,
+              "strats": [
+                {
+                  "name": "Green Pirates and Mini-Kraid Farm",
+                  "notable": false,
+                  "requires": [
+                    {"resetRoom": {
+                      "nodes": [1, 2],
+                      "mustStayPut": false
+                    }},
+                    {"or": [
+                      "Super",
+                      "Ice",
+                      "Wave",
+                      "Spazer",
+                      "Plasma",
+                      {"and": [
+                        "Bombs",
+                        "canCarefulJump"
+                      ]}
+                    ]},
+                    {"refill": ["Energy", "Missile", "Super"]}
+                  ]
+                },
+                {
+                  "name": "Mini-Kraid Farm",
+                  "notable": false,
+                  "requires": [
```

Comment:
mini kraid should have some requirements.
2 spike hits, or supers, or missiles, or speed, or beams.  There are a lot of options, but with nothing you'll get hit first.

Reply:
I took another pass this. Got rid of the separate Mini-Kraid farm and added those requirements to the combined farm. Removed the option of using Bombs to kill the Pirates since that is not fun. With the slow methods (everything except Screw, Plasma, or Super), assumed either a careful jump or a stone hit from Mini-Kraid; spike hit is also possible but just wanted to include the worst case.